jcommune-model/src/test/java/org/jtalks/jcommune/model/entity/JCUserTest.java

```diff
@@ -217,6 +220,37 @@ public void spanishLanguageHandling () {
         assertEquals(language, Language.ENGLISH);
     }
 
+    @Test
+    public void entityFieldsShouldBeSerialized() throws NoSuchMethodException, InvocationTargetException,
+            IllegalAccessException {
+        JCUser user = ObjectsFactory.getUserWithAllNotInheritedFieldsFilled();
+        user.setAvatar(new byte[]{1});
```

Comment:
I reasoned thus these fields are the part of `jtalks-common` API and wondered would it be really right approach to set these ones at the jcommune' `ObjectFactory`, (moreover there are needs to use reflection to set up some fields)?
If this is ok I'll fill all these fields at `ObjectFactory`